builtin/providers/google/resource_google_project_services.go

```diff
@@ -160,7 +168,9 @@ func getApiServices(pid string, config *Config) ([]string, error) {
 		return apiServices, err
 	}
 	for _, v := range svcResp.Services {
-		apiServices = append(apiServices, v.ServiceName)
+		if _, ok := ignore[v.ServiceName]; !ok {
+			apiServices = append(apiServices, v.ServiceName)
+		}
```

Comment:
I could be wrong on this, but I think we could use a DiffSuppressFunc instead of modifying the state for this. It's not a huge deal, but in the unlikely event the API starts letting us control one of these services in the future, Terraform would know it's already enabled instead of thinking it's not yet. What do you think?